AxonDeepSeg/apply_model.py

```diff
@@ -227,9 +227,26 @@ def axon_segmentation(path_acquisitions_folders, acquisitions_filenames, path_mo
 
     # If we did not receive any resolution we read the pixel size in micrometer from each pixel.
     if acquired_resolution == 0.0:
-        resolutions_files = [open(os.path.join(path_acquisition_folder, 'pixel_size_in_micrometer.txt'), 'r')
-                             for path_acquisition_folder in path_acquisitions_folders]
-        acquisitions_resolutions = [float(file_.read()) for file_ in resolutions_files]
+
+        if os.path.exists(os.path.join(path_acquisitions_folders[0], 'pixel_size_in_micrometer.txt')):
+
+            resolutions_files = [open(os.path.join(path_acquisition_folder, 'pixel_size_in_micrometer.txt'), 'r')
+                                for path_acquisition_folder in path_acquisitions_folders]
+
+            acquisitions_resolutions = [float(file_.read()) for file_ in resolutions_files]
+
+
+        else:
+
+            tmp_path, selected_model = os.path.split(path_model_folder)
+
+            if selected_model == "default_TEM_model_v1":
+                acquisitions_resolutions = 0.01
```

Comment:
no, we should not set a default for this, because i know that 50% of our users will ignore the warning, have wrong results, publish with it, and 2 years later someone will realize that the results were wrong.